README.md

```diff
@@ -21,7 +26,66 @@ containing ascii characters):
 assert(scopeUtils.validScope("..."));
 ```
 
-### Checking Scope Sets
+### Scope Expressions
+
+These are the "new-style" way of dealing with scopes and allow for greater
+flexibility than the old style. The expressions are objects with a single key --
+either `AnyOf` or `AllOf`. The value must be an array and the elements of the
+array must be either expressions or a [valid scope](#valid-scopes).
```

Comment:
So, strings are not valid at the top level -- that is, `"my-service:some-string"` is not a valid expression.